backend/models/admin.js

```diff
@@ -0,0 +1,25 @@
+const mongoose = require('mongoose')
+
+const adminSchema = new mongoose.Schema({
+    firstName: {
+        type: String,
+        required: true
+    },
+    lastName: {
+        type: String,
+        required: true
+    },
+    email: {
+        type: String,
+        required: true,
+        unique: true
+    },
+    password: {
+        type: String,
+        required: true,
+        match: [/\S+@\S+.\S+/, 'is invalid']
```

Comment:
Could you actually move this match statement up to email instead of password, I realized I messed this up in the description and the email should have the @ symbol not password 😅